src/main/java/org/sasanlabs/internal/utility/annotations/VulnerableAppRequestMapping.java

```diff
@@ -22,6 +24,13 @@
     @AliasFor(annotation = RequestMapping.class)
     String value();
 
+    /**
+     * Specify whether the implementation can be considered secure, as in, non-exploitable.
+     *
+     * @return variant
+     */
+    String variant() default VariantConstants.UNSECURE;
```

Comment:
Please go ahead with Enum.

Reply:
can we name it as SecurityVariant or SecureVariant ?

Reply:
I've got a mild preference for the current approach as I believe this _security_ part might be inferred by looking at sibling and parent properties. However, if you feel like your approach could avoid confusion in the future, I'm happy to make an update. Please let me know your final decision.

Reply:
then lets go with variant only but add a javadoc explaining the use of this field